lemur/domains/views.py

```diff
@@ -172,6 +173,7 @@ def get(self, domain_id):
         return service.get(domain_id)
 
     @validate_schema(domain_input_schema, domain_output_schema)
+    @operator_permission.require(http_exception=403)
```

Comment:
this is gated by admin already https://github.com/Netflix/lemur/blob/1c096f1028a237a6f58d181e5aaabb0f8f1e0953/lemur/auth/permissions.py#L25